data/Spectacole-Actori.csv

```diff
@@ -0,0 +1,5 @@
+1,1,2,3
+2,4,5,6
+3,7,8,9,10,11,12,13
+4,14,15,16
+5,17,18,19,20
```

Comment:
nu este foarte clar ce reprezinta fiecare valoare, acest csv este specific nu unei singure entitati din cate deduc, ci doresti sa faci relatia dintre 2 entitati, adica ce actori sunt in fiecare spectacol. ar ajuta astfel sa ai si prima linie in csv cu denumirea field-ului pentru a fi usor de parcurs in cazul asta, asta ca si sugestie